ntc_templates/templates/cisco_nxos_show_ip_route.textfsm

```diff
@@ -45,6 +45,7 @@ Start
   ^'\S+'\s+denotes\s
   ^'%<string>'
   ^\s*$$
+  ^\s*via.*
```

Comment:
You can get rid of this and change L32 to the following:
`  ^\s+\*?via ${NEXTHOP_IP}(%${NEXTHOP_VRF})?, ${NEXTHOP_IF}, \[${DISTANCE}/${METRIC}\], ${UPTIME}, ${PROTOCOL}, ${TYPE}(, tag ${TAG})? -> Record`

Instead of having a `+` after `. ^\s+\*` we can either add another asterick or the `?` to denote 0 or 1